index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Website</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+          <a>My Logo</a>
+        </div>
+        <nav class="nav-bar">
+            <div class="ham-menu">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg1">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg2">
+            <img src="image/Vector.png" alt="Nav Bar" class="navImg3">
+        </div>
+        </nav>
+    </header>
+    <main>
+       <section>
+           <div class="Backgroud">
+  
+    
+           </div>
+        <div class="title-name">
+            <h1>I’m Amol<br>
+                Glad to see you!
+            </h1>
+        </div>
+        <div class="title-intro">
+            I’m a software developer! I can help you build a product , feature or website
+                 Look through some of my work and experience! If you like what you see 
+                 and have a project you need coded, don’t hestiate to contact me.
+            
+        </div>
```

Comment:
- [ ] Optional: Highly suggested to implement this. As suggested previously you don't need a div tag but you have removed p tags. Make sure you have p tags and replace div tags with p tags.